components/library/MediaInfoLib/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022, Daniel Chan
+#
+
+BUILD_BITS=64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		MediaInfoLib
+COMPONENT_VERSION_DL= 21.09
+COMPONENT_VERSION=	21.9
+COMPONENT_SUMMARY=	MediaInfo(Lib) is a convenient unified display of the most relevant technical and tag data for video and audio files.
```

Comment:
You could also add HUMAN_VERSION=21.09 (and use it in the package manifest). This way pkg info will also show the correct version information. See sysutils/tmux for an example usage.